src/main/java/com/future/function/qa/util/Cleaner.java

```diff
@@ -61,4 +64,9 @@ private void setMarkUsedFalse(String collectionName, String id) {
       .updateOne(new BasicDBObject("_id", id), Updates.set("used", false));
   }
 
+  public void hardDeleteMessageStatus(String messageId) {
+    mongoDatabase.getCollection(DocumentName.MESSAGE_STATUS)
+            .deleteMany(new BasicDBObject("message.$id", ObjectId.isValid(messageId) ? new ObjectId(messageId) : messageId));
```

Comment:
Sorry, what is the business flow here? Why delete message status by message id?

Reply:
For every create message api called, some message status also will be created. I cant get the message status id related to that message, so i need to remove the message status by using message id.

Reply:
Ok, but a message cannot be deleted by user, right?

Reply:
yep